pkg/telemetryservice/sql/mysql/mysql.go

```diff
@@ -41,7 +41,7 @@ func (db *DBHelper) SendToDB(ctx context.Context, deviceName string, rawData []b
 		return err
 	}
 
-	return nil
+	return db.DB.Close()
```

Comment:
`defer db.DB.Close()` if no error at line 37?

Reply:
Defer does solve the problem, but it doesn't easily handle errors, and the direct return close method makes the code more concise.

Reply:
Wait a minute I ask gpt

Reply:
ok defer is better thanks for your comment tom😄